src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
+
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final int FIRST_INDEX = 0;
+    private static final int SECOND_INDEX = 1;
+    private static final String SUPPLY_PATTERN = "supply";
+    private static final String BUY_PATTERN = "buy";
+    private static final String SPLIT_PATTERN = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataFromFile = readFile(fromFileName);
+        writeFile(toFileName, createReport(dataFromFile));
+    }
+
+    private String[] readFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        String[] context = null;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            int value = bufferedReader.read();
```

Comment:
Why not use bufferedReader.readLine() here to read the string?

Reply:
> Why not use bufferedReader.readLine() here to read the string?

Okay, I'll change it. Thanks for variant.